src/main/java/com/bloxbean/cardano/client/util/JsonUtil.java

```diff
@@ -32,4 +33,8 @@ public static String getPrettyJson(String jsonStr) {
     public static JsonNode parseJson(String jsonContent) throws JsonProcessingException {
         return mapper.readTree(jsonContent);
     }
+
+    public static String[] usingSplitMethod(String text, int n) {
```

Comment:
If I understand correctly, this method is not specific to json. Should we create a new util for string operations and add to that? Also please add a test for this new method